android/app/src/main/java/com/zulipmobile/notifications/NotificationHelper.java

```diff
@@ -65,22 +66,22 @@ public static String extractName(String key) {
         return key.split(":")[0];
     }
 
-    public static void buildNotificationContent(LinkedHashMap<String, Pair<String, Integer>> conversations, Notification.InboxStyle inboxStyle, Context mContext) {
-        for (Map.Entry<String, Pair<String, Integer>> map : conversations.entrySet()) {
+    public static void buildNotificationContent(LinkedHashMap<String, List<MessageInfo>> conversations, Notification.InboxStyle inboxStyle, Context mContext) {
+        for (Map.Entry<String, List<MessageInfo>> map : conversations.entrySet()) {
             String name = extractName(map.getKey());
 
             Spannable sb = new SpannableString(String.format(Locale.ENGLISH, "%s%s: %s", name,
-                    mContext.getResources().getQuantityString(R.plurals.messages, map.getValue().second, map.getValue().second),
-                    map.getValue().first));
+                    mContext.getResources().getQuantityString(R.plurals.messages, map.getValue().size(), map.getValue().size()),
```

Comment:
(This is for a followup, because it's pre-existing)
This repeated `map.getValue()` would be clearer... well, first by saying `entry`, but then by giving this expression a name:
```
List<MessageInfo> messages = entry.getValue();
```
I'd put that right at the top of the loop, just under the `...entry.getKey()` so the two of them are immediately under the `for` line.